pkg/restore/restore.go

```diff
@@ -696,10 +798,15 @@ func (ctx *context) restoreResource(resource, namespace, resourcePath string) (a
 
 			// Check if the PV exists in the cluster before attempting to create
 			// a volume from the snapshot, in order to avoid orphaned volumes (GH #609)
-			_, err := resourceClient.Get(name, metav1.GetOptions{})
+			shouldRestoreSnapshot, err := ctx.waitForPVDeletion(name, resourceClient)
```

Comment:
I think it would make the logic more clear if method and variable were both named the same, whichever makes the most sense. Either `waitForPVDeletion` or `shouldRestoreSnapshot`.

Reply:
I'm not sure about naming the method and variable the same name; @skriss or @ncdc thoughts?

Reply:
I'll change the names to match and push another revision up.